### PR TITLE
Hero section component

### LINKED ISSUE #2: Reusable hero section component

### PR DESCRIPTION
**Pull request by mustafaguldag**
I did build Hero Section component. #2

### CODE REVIEW

**Review by ayse8888 — APPROVED** [review]
i checked the code and it looks reusable and clean. there is no unnecessary spaces in the code.

File: src/components/HeroSection/HeroSection.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  background-position: center;
+  background-size: cover;
+  height: 55vh;
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.findmebutton {
```
Comment (halitbatur) [inline src/components/HeroSection/HeroSection.css]: class names should be camelCase

File: src/components/HeroSection/HeroSection.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.container h2 {
+  color: #ff3e3e;
+}
+
+.container p {
+  color: #777777;
+}
+
+.buttonlogo {
```
Comment (halitbatur) [inline src/components/HeroSection/HeroSection.css]: camelCase

File: src/components/HeroSection/HeroSection.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import logo from "../../images/buttonlogo.png";
+import "./HeroSection.css";
+
+const HeroSection = ({ img, title, text }) => {
+  const background = {
+    backgroundImage: `url(${img})`,
+  };
+
+  return (
+    <Container className="container" style={background}>
```
Comment (halitbatur) [inline src/components/HeroSection/HeroSection.js]: this component does not handle showing the title,text and the button on the right

**Review by halitbatur — CHANGES_REQUESTED** [review]
looks good overall but need some fixes

File: src/components/HeroSection/HeroSection.css (hunk 1 above)
Comment (mustafaguldag) [reply src/components/HeroSection/HeroSection.css]: Of Course, I am on it. Fixing it. Thanks Halit

File: src/components/HeroSection/HeroSection.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <Container className={changeLeftOrRight()} style={background}>
+      <Row>
+        <Col sm="8">
+          <h2>{title}</h2>
+          <p>{text}</p>
+          <a href="/search">
+            <button className="findMeButton">
+              <img className="buttonLogo" src={logo} />
```
Comment (halitbatur) [inline src/components/HeroSection/HeroSection.js:27]: image src needs to be dynamic here

File: src/components/HeroSection/HeroSection.test.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import HeroSection from "./HeroSection.js";
+import React from "react";
+import renderer from "react-test-renderer";
+
+it("renders correctly", () => {
+  const tree = renderer.create(<HeroSection />).toJSON();
```
Comment (halitbatur) [inline src/components/HeroSection/HeroSection.test.js]: please pass some props to the component in order to see if it renders things correctly or not

**Review by halitbatur — CHANGES_REQUESTED** [review]
2 more changes and I think its ready to be merged, good work!

File: src/components/HeroSection/HeroSection.js (hunk 4 above)
Comment (mustafaguldag) [reply src/components/HeroSection/HeroSection.js:27]: Thanks for review Halit. This button will be same in our component and never change thats why I didnt make dynamic, still do you want me to make it dynamic ?

File: src/components/HeroSection/HeroSection.test.js (hunk 5 above)
Comment (mustafaguldag) [reply src/components/HeroSection/HeroSection.test.js]: I will do. Thanks

File: src/components/HeroSection/HeroSection.js (hunk 4 above)
Comment (halitbatur) [reply src/components/HeroSection/HeroSection.js:27]: @mustafaguldag its okai then

**Review by halitbatur — APPROVED** [review]

**Pull request merged**